assets/messages.yml

```diff
@@ -209,6 +209,23 @@ messages:
         The authentication services are offline.
         This will be fixed shortly, please hang tight. Check tweets from [@MojangSupport|https://twitter.com/MojangSupport] for the current status.
 
+        %quick_links%
+      fillname: []
+  banned:
+    - project:
+        - mcd
+        - mcpe
+        - realms
+        - web
+      name: Banned
+      shortcut: banned
+      message: |-
+        *Thank you for your report!*
+        However, this issue is {color:#FF5722}*Invalid*{color}.
+
+        This is an account. We do not have the tools to help you with this issue on this bug tracker.
```

Comment:
"This is an account issue. (...)". Without referring to an account _issue_, it'll read weird. The "this" refers to the ticket, and a ticket is not an account.